stratum/lib/security/BUILD

```diff
@@ -203,27 +201,29 @@ stratum_cc_library(
 )
 
 stratum_cc_library(
-    name = "cert_utils",
+    name = "certificate",
+    # TODO(bocon): seed random number generator before using this in production
+    # code
```

Comment:
By default, OpenSSL on Linux should seed the RNG automatically from `/dev/urandom` the first time bytes are need to generate an RSA keypair. This is definitely sufficient for testing, and probably sufficient for production use cases.

See https://wiki.openssl.org/index.php/Random_Numbers#Initialization for more details on OpenSSL seeding.

Per the (man page)[https://man7.org/linux/man-pages/man4/random.4.html],
```
The /dev/random interface is considered a legacy interface, and
       /dev/urandom is preferred and sufficient in all use cases, with
       the exception of applications which require randomness during
       early boot time; for these applications, getrandom(2) must be
       used instead, because it will block until the entropy pool is
       initialized.
```

Reply:
For context, that's your comment I moved from the [header](https://github.com/stratum/stratum/pull/876/files#diff-46a71815adfa8f084e63ee53e5ae319baa327104993409a9d6d3fb1d39ec23eeL65-L66) to the Bazel rule that a future dev will change to use this lib in non-test code.

If you think it's solved, we can remove the comment and `testonly` restriction.

Reply:
Yup, got it. And, here is a better place for the comment, so thanks for moving it.

Regarding whether this is solved, I think it largely is. I'm not sure what "early" means in the man page, but I think that by the time the system boots, Docker loads, etc., there should be sufficient entropy to generate a short-lived machine-to-machine communication key pair if someone wanted to do that

I think we drop the `testonly` restriction if that's okay with you.

Reply:
removed